src/components/StudyLegend.jsx

```diff
@@ -32,6 +33,7 @@ class StudyLegend extends Component {
                         className={`ic-icon-with-sub ${menuOpen ? 'active' : ''}`}
                         tooltip-title={t.translate('Indicators')}
                     />
+                    {activeStudiesNo ? <span className={`budget ${activeStudiesNo > 9 ? 'x2' : ''}`}>{activeStudiesNo}</span> : ''}
```

Comment:
the same code is duplicated in `Comparison.jsx`. Consider turning this to a component. 

Also why do you use the name `budget`? It's called a notification badge.

Reply:
@brucebinary As Afiq mention on the Trello card, width for 1-digit number should be `16px` and for 2-digit should be `24px`. That's why I use `activeStudiesNo > 9 ? 'x2' : ''`

Reply:
> the same code is duplicated in Comparison.jsx. Consider turning this to a component.

@shayanbinary I'm not talking about the width of the badge.

Reply:
@brucebinary so, I don't get you about this

Reply:
@shayanbinary create a `<NotificationBadge />` component